### ISSUE TITLE
Holding the Atlas in either the main or off hand both removes the crosshair.

### ISSUE THREAD

**Issue by NotoriousLemon** [Hunternif/AntiqueAtlas#331]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Forge version information (please complete the following information):**
 - Minecraft: [e.g. 1.16.4]
 - Forge: [e.g. 35.1.37]
 - AntiqueAtlas: [e.g. 5.4.2-forge]

**Fabric version information (please complete the following information):**
 - Minecraft: [e.g. 1.16.5]
 - Fabric Launcher: [e.g. 0.11.1]
 - Fabric API: [e.g. 0.29.4]
 - AntiqueAtlas: [e.g. 5.4.2-fabric]

**Additional context**

**Comment by NotoriousLemon** [comment]
The atlas is also not displayed in hand. I am unsure what causes this bug. I previously had the portal atlas mod installed but i removed that loaded the world and the error persists, evidently removing the mod replaces the portal markers with X's. Normally the atlas is displayed while holding and normally the crosshair is a permanent part of the hud however this does not appear to be the case. I opened a different world with the same modset and there was no problem. This probably only started occurring after several hours in my survival world. I made another atlas and the problem persisted.
![image](https://user-images.githubusercontent.com/80377450/110580558-900a3980-812e-11eb-915c-7c1f79b3c0f2.png)
![2021-03-09_22 55 45](https://user-images.githubusercontent.com/80377450/110580625-a9ab8100-812e-11eb-972b-591a66d9e89c.png)
![2021-03-09_22 55 47](https://user-images.githubusercontent.com/80377450/110580631-add79e80-812e-11eb-8820-4e7a633e162e.png)
